test/integration/fixture_image_simple_test.go

```diff
@@ -190,7 +190,8 @@ func BenchmarkSimpleImage_FetchSquashedContents(b *testing.B) {
 
 func assertImageSimpleMetadata(t *testing.T, i *image.Image, expectedValues testCase) {
 	t.Helper()
-	t.Log("Asserting metadata...")
+	//t.Log("Asserting metadata...")
```

Comment:
nit: this is commented out in the test - do we want to keep this or delete it?

Reply:
good question -- I think I'll keep it for now